Source/Gradient.swift

```diff
@@ -9,6 +9,7 @@ public struct GradientComponents {
 
 public protocol Gradient {
 
+    var endPoint: CGPoint { get }
```

Comment:
This is a great catch, thanks!

Reply:
Building on this, I think it'd be good to use `GradientPoints` instead of having 2 separate properties for `startPoint` and `endPoint` in the `Gradient` protocol to ensure that others don't make a similar mistake. What do you think?

Reply:
Great idea